cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -46,6 +46,7 @@ if [ -n "${UNMANAGED_INSTALL}" ]; then
     NO_MODIFY_PATH=1
     INSTALL_UPDATER=0
 fi
+AUTH_TOKEN="{{ '${' }}{{ env_vars.github_token_env_var }}:-}"
```

Comment:
Could this be
```
AUTH_TOKEN="${{{ env_vars.github_token_env_var }}:-}"
```

Reply:
iirc no because `${{` is part of the jinja syntax we have enabled in this file

Reply:
Are you sure `${{` is jinja syntax? I can't find it (but I'm no expert)

Reply:
Right yeah sorry I'm remembering release.yml.j2 where github uses `${{ }}` and `{{ }}` (in yml we customize jinja to use `{{{ }}}` instead of `{{ }}` so it's not ambiguous).

In sh the issue is ${{{ gets greedily parsed as `$` `{{` `{` instead of `${` `{{`, and the templates try really hard to have clean/nice output for reading/linting so adding fudging whitespace isn't desirable.

Reply:
ah ok